integration-tests/src/test/java/oracle/weblogic/kubernetes/ItWlsSamples.java

```diff
@@ -70,7 +70,7 @@
 @TestMethodOrder(MethodOrderer.OrderAnnotation.class)
 @DisplayName("Verify the domain on pv, domain in image samples using wlst and wdt and domain lifecycle scripts")
 @IntegrationTest
-class ItWlsSamples {
+public class ItWlsSamples {
```

Comment:
Can you please remove the public from here and also all the test methods?
Ryan merged https://github.com/oracle/weblogic-kubernetes-operator/pull/2492 to make this change and you are overwriting this change.